### ISSUE TITLE
Domino CSS intrusive to existing applications

### ISSUE THREAD

**Issue by jhickman** [DominoKit/domino-ui#613]
I have experimenting with introducing Domino-UI to an existing application (7+ years application age).  However, just importing the css through the following `<link>` causes a lot of application rendering/layout issues.
```xml
<link type="text/css" rel="stylesheet" href="{module-short-name}/css/domino-ui.css"> 
<link type="text/css" rel="stylesheet" href="{module-short-name}/css/themes/all-themes.css">
```

From quick inspection of the domino CSS, I find that there are several places that use CSS tag selectors rather than class selectors such as:
```css
html { /* ... */ }
body { /* ... */ } 
table { /* ... */ }
/* etc */
```

I have also found that there are several CSS classes that are currently in use in the application, such as `.btn`.   I haven't done a thorough inspection to see any others that may be in conflict.

---
My proposed solution would be to always use class selectors rather than tag selectors to prevent being intrusive to existing applications that may already have a "reset.css" or similar and rely upon inherited styles of tags.

For conflicting class names, my suggestion is to either use obfuscated names (similar to those seen with ClientBundle/CssResource) or namespace the classes with some kind of prefix such as `.dui-btn`, `.dk-btn`, etc.

---

As an alternative, I have considered just not using the Domino CSS, and only copy/rewrite the portions needed for used widgets, but that task would be very time consuming.

---

Here are a couple of screenshots of before/after importing the domino CSS in just a small portion of the application window.

Before:
![image](https://user-images.githubusercontent.com/861908/144623653-282d5184-998c-4d57-9d2c-664e26b43e51.png)

After:
![image](https://user-images.githubusercontent.com/861908/144623747-153a14e2-c4c3-45d8-affd-fd7734add539.png)

**Comment by vegegoku** [comment]
this is planned for the next milestone which is the milestone of themeing.

**Comment by vegegoku** [comment]
This is fixed in version 2.0.0-RC1 onward.
